File: Backend/src/main/java/com/jfrb/POCSucripcion/model/Usuario.java
package com.jfrb.POCSucripcion.model;

import jakarta.persistence.Entity;
import jakarta.persistence.Id;
import jakarta.persistence.Table;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.time.LocalDateTime;

@Data
@NoArgsConstructor
@AllArgsConstructor
@Entity
@Table(name = "USUARIO")
public class Usuario {

    @Id
    private int codigo;

    private Long numDocumento;
    private String tipoDoc;
    private String nombre;
    private String email;
    private LocalDateTime fechaRegistro;
    private String estado;
}

File: Cliente1 VS/POCSuscripcionCliente/Models/Usuario.cs
using System;

namespace POCSuscripcionCliente.Models
{
    public class Usuario
    {
        public int codigo { get; set; }
        public long numDocumento { get; set; }
        public string tipoDoc { get; set; }
        public string nombre { get; set; }
        public string email { get; set; }
        public DateTime? fechaRegistro { get; set; }
        public string estado { get; set; }

        public string EstadoTexto
        {
            get
            {
                return estado == "AC" ? "Activo" : "Inactivo";
            }
        }
    }
}